orion/pipelines/verified/lstm_dynamic_threshold/lstm_dynamic_threshold.json

```diff
@@ -25,6 +26,11 @@
         },
         "keras.Sequential.LSTMTimeSeriesRegressor": {
             "epochs": 35
+        },
+	"orion.primitives.timeseries_anomalies.find_anomalies#1": {
+            "window_size_portion": 0.33,
+            "window_step_size_portion": 0.1,
+            "fixed_threshold": true
```

Comment:
I am wondering for dynamic_threshold one, we should use false for fixed_threshold?

And we can create another one called "lstm.json" which use `true` value the fixed_threshold for benchmarking purpose?

Reply:
Good suggestion, let's think about a design here:
* two pipelines:
    * lstm_dt -- dynamic threshold
    * lstm -- fixed threshold

Technically, they both use the same "template" but different hyperparameter settings. So we can have them as two separate pipelines or just one pipeline with multiple hyperparameter settings.

For verbosity, I prefer option 1 but it will have a lot of redundancy.

Reply:
I understand. Based on what we implemented for the automatic benchmarking, option 1 is the easy solution.

Reply:
See issue #204.